### PR TITLE
[ja] add: translation of contributing

### PR DESCRIPTION
**Pull request by kaitoii11**
translate `content/docs/contributing/_index.md` to Japanese.r

### CODE REVIEW

File: content/ja/docs/contributing/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: コントリビューション
+description: OpenTelemetryドキュメントへのコントリビューション方法について学ぶ。
+aliases: [/docs/contribution-guidelines]
+weight: 980
+default_lang_commit: a4f4d8b07313532d44d4041183c515875ad4d60d
+---
+
+OpenTelemetryドキュメントコントリビューターは以下を行ないます:
```
Comment (ymotongpoo) [inline content/ja/docs/contributing/_index.md:9]: nit: please avoid using colons in Japanese; you could use normal period (句点) here.

File: content/ja/docs/contributing/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- 既存コンテンツの改善。
+- 新しいコンテンツの作成。
+- OpenTelemetry Registryの更新。
+- サイトをビルドするコードの改善。
+
+以下の手引はOpenTelemetryドキュメントへのコントリビュート方法について説明します。
+
+OpenTelemetryプロジェクトへの一般的なコントリビュートの手引には、[OpenTelemetryコントリビュートガイド](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
+を参照ください。
+コントリビュートライセンス規約(Contributor License Agreement)と行動規範(Code of Conduct)の詳細が記されています。
```
Comment (ymotongpoo) [inline content/ja/docs/contributing/_index.md:20]: this is just a confirmation, but is it common to translate "Contributor" to "コントリビュート"? I see that consistency in your translation, I'm wondering which is better to translate "コントリビュート" or "コントリビューター" here, though the impressions of both translation give are mostly same.

**Review by ymotongpoo — COMMENTED** [review]
Thank you for your contribution! I left some comments and I'd like to hear your thoughts on those.

File: content/ja/docs/contributing/_index.md (hunk 2 above)
Comment (kaitoii11) [reply content/ja/docs/contributing/_index.md]: Typo for the コントリビュートライセンス規約. I fixed it to コントリビューターライセンス規約 to match the original text.

File: content/ja/docs/contributing/_index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+OpenTelemetryドキュメントコントリビューターは以下を行ないます。
+
+- 既存コンテンツの改善。
+- 新しいコンテンツの作成。
+- OpenTelemetry Registryの更新。
+- サイトをビルドするコードの改善。
+
+以下の手引はOpenTelemetryドキュメントへのコントリビュート方法について説明します。
+
+OpenTelemetryプロジェクトへの一般的なコントリビュートの手引には、[OpenTelemetryコントリビューティングガイド](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)を参照ください。
```
Comment (ymotongpoo) [inline content/ja/docs/contributing/_index.md:18]:
```suggestion
OpenTelemetryプロジェクトへの一般的なコントリビュートの手引には、[OpenTelemetryコントリビューターガイド](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)を参照ください。
```

**Review by ymotongpoo — CHANGES_REQUESTED** [review]
the doc linked is named "OpenTelemetry Contributor Guide" so I fixed here. otherwise, all LGTM, so I'll approve this PR as soon as the suggestion is merged.

**Review by ymotongpoo — APPROVED** [review]
LGTM

**Comment by svrnm** [comment]
Thank you @kaitoii11 !

**Review by svrnm — APPROVED** [review]

**Pull request merged**